e2e/test/Helpers/logging/MsTestLogger.cs

```diff
@@ -30,10 +30,5 @@ private MsTestLogger(TestContext testContext) : base()
             Properties.Add(LoggingPropertyNames.TargetFramework, targetFramework.FrameworkName);
             Properties.Add(LoggingPropertyNames.OsInfo, operatingSystem);
         }
-
-        public static MsTestLogger GetInstance(TestContext testContext)
```

Comment:
This does not need to be removed, I had modifies this as a part of my internal testing, and forgot to revert.